test/execute_noCLI.test.js

```diff
@@ -0,0 +1,195 @@
+const assert = require('assert');
+const Execute_noCLI = require('../lib/execute_noCLI');
+
+describe("Execute_noCLI", () => {
+    it("expects an expression", () => {
+        assert.throws(
+            () => { Execute_noCLI() },
+            /Cannot execute without an expression./
+        )
+    })
+
+    it("expects an initial state", () => {
+        assert.throws(
+            () => { Execute_noCLI({ expression: "foo" }) },
+            /Cannot execute without an initial state./
+        )
+    })
+
+    describe("when given an expression and state", () => {
+        it("returns", async () => {
+            let result = await Execute_noCLI(
+                `
+                alterState(state => {
+                    return state.num + 1
+                 })`,
+                {"num": 1},
+                'language-dhis2'
```

Comment:
Does this need to be required? Ideally, we'd have a new test in the `language-dhis2` adaptor (rather than in here) that attempted to work with this version of `core`.

Reply:
I agree with you, here I only included it because I thought javascript would throw an error for not including the language parameter, but I see it can be left empty so I will adapt it accordingly, thank you!